### ISSUE TITLE
GDAX cancel_order exception in PHP

### ISSUE THREAD

**Issue by MrDiscret** [ccxt/ccxt#615]
First of all, thank you for the work you've been putting into this project!

I'm looking/testing the GDAX php functionality and found out that the cancel_order method doesn't seem to be working, or... I'm doing something wrong. Using the GDAX sandbox. 

All other methods I tested so far (everything that's above the cancel_order in the gdax.php file) seem to be working fine.

After I successfully created a CDAX order using create_order, I retreived the id of that order and tried to cancel it using:

```
cancel_order('7bb61f78-51fe-42b1-adae-e6659ade7578')
```

where `7bb61f78-51fe-42b1-adae-e6659ade7578` is the ID of the previously created, existing order.

The response I'm getting is:

```
Fatal error:  Uncaught exception 'ccxt\ExchangeError' with message 'gdax {"message":"invalid signature"}' in C:\...\trade\ccxt\php\gdax.php:500
Stack trace:
#0 C:\...\trade\ccxt\php\base\Exchange.php(731): ccxt\gdax->handle_errors(400, '', 'https://api-pub...', 'DELETE', Array, '{"message":"inv...')
#1 C:\...\trade\ccxt\php\base\Exchange.php(606): ccxt\Exchange->fetch('https://api-pub...', 'DELETE', Array, NULL)
#2 C:\...\trade\ccxt\php\gdax.php(507): ccxt\Exchange->fetch2('orders/{id}', 'private', 'DELETE', Array, NULL, NULL)
#3 [internal function]: ccxt\gdax->request('orders/{id}', 'private', 'DELETE', Array)
#4 C:\...\trade\ccxt\php\base\Exchange.php(549): call_user_func(Array, 'orders/{id}', 'private', 'DELETE', Array)
#5 [internal function]: ccxt\Exchange->ccxt\{closure}(Array)
#6 C:\...\trade\ccxt\php\base\Exchange.php(1307): call_user_func_array(Object(Closure), Array)
#7 C:\... in C:\...\trade\ccxt\php\gdax.php on line 500
```

Help would be greatly appreciated.

**Comment by kroitor** [comment]
@MrDiscret hi! Thanks for reporting this issue! We will will take a look into it and will get back to you with a fix asap!

**Comment by MrDiscret** [comment]
I have selected all permissions (view, trade, manage, transfer, bypass-2fa) when generating the API key.
Also tried a new API key but that didn't solve the problem.

Just thought it was worth mentioning...

**Comment by kroitor** [comment]
@MrDiscret thank you for the feedback, it can really make a difference sometimes, indeed. I'm fixing this, will be there soon and will let you know, of course.

**Comment by MrDiscret** [comment]
@kroitor I hope my comments will not be confused with trying to be pushy :)
Just providing some information that may rule out some possibilities...

So I just tried the same time using the live (real) GDAX API but the result was the same.

**Comment by kroitor** [comment]
@MrDiscret I totally understand, the more comments – the better, feel free to add whatever you think is important or worth noting. I really appreciate your help and your detailed reporting, it does make the job a little easier for me. This is a generic error with GDAX cancel_order in PHP and it will behave the same within their sandbox and live. I'll fix this, hold on, please )

**Comment by kroitor** [comment]
This issue was fixed in version 1.10.170 of ccxt. Don't forget to update. We will be happy, if you report whether it works for you or not, tested it locally on our side and it worked. Thx for the feedback! Cheers!

**Comment by MrDiscret** [comment]
I've been off for the last few days thus my late reply.

The cancel_order function **now works** with the live GDAX API.

I first tried to use the gdax sandbox several times and it did not work (Fatal error:  Uncaught exception 'ccxt\RequestTimeout' with message 'gdax DELETE https://api-public.sandbox.gdax.com/orders/97326cf6-f3a4-4f00-b16d-a18ca124d45a 28 Operation timed out after 10000 milliseconds with 0 bytes received')
That's when I thought I'd give it a try using the real (or live, as I sometimes call it) API and it did go trough and cancelled one of the open orders I had.

@kroitor Thanks a lot!
